report_thesis/src/sections/introduction.tex

```diff
@@ -1,26 +1,34 @@
 
-Part of this research is facilitated through interpretation of spectral data gathered  by \gls{libs} instruments.
-\gls{libs} works by firing a high-powered laser at soil samples, creating a plasma that emits light captured by spectrometers and analyzed for its elemental composition through the use of machine learning models.
-Scientists and researchers at NASA uses the spectral data to inform their understanding of Mars' geology through the identification of major oxides in the soil samples.
-To this end, various machine learning models have been developed to predict the composition of major oxides in the sample.
-However, the high dimensionality and multicollinearity of the spectral data remains a significant challenge for these models.
-A detailed study into advanced machine learning models and dimensionality reduction techniques is therefore crucial for improving predictions in these applications.
+Part of this research is facilitated through interpretation of spectral data gathered by \gls{libs} instruments, which fire a high-powered laser at soil samples to create a plasma.
+The emitted light is captured by spectrometers and analyzed using machine learning models to assess the presence and concentration of certain major oxides, informing NASA's understanding of Mars' geology.
 
+However, predicting major oxide compositions from \gls{libs} data presents significant computational challenges, including the high dimensionality and non-linearity of the data, compounded by multicollinearity and matrix effects~\cite{andersonImprovedAccuracyQuantitative2017}. 
+These effects can cause the intensity of emission lines from an element to vary independently of that element's concentration, introducing unknown variables that complicate the analysis. 
+Furthermore, due to the high cost of data collection, datasets are often small, which further complicates the task of building accurate and robust models.
 
-In this work, we show that... \textit{results of the study will be added here after the paper is written.}
+Various machine learning models have been used to predict the composition of major oxides in the sample, including \glspl{cnn}~\cite{yang_laser-induced_2022, yangConvolutionalNeuralNetwork2022}, \gls{svr}~\cite{rezaei_dimensionality_reduction}, and hybrid models like \gls{df}-\gls{k-elm}~\cite{song_DF-K-ELM} that incorporate domain knowledge to enhance model interpretability and performance.
+However, the high dimensionality and multicollinearity of the spectral data remains a significant challenge for these models.
 
-Our experimental results demonstrate that... \textit{results of the experiment will be added here after the paper is written.}
+Building upon the baseline established in~\citet{p9_paper}, this thesis aims to address the challenges in predicting major oxide compositions from \gls{libs} data by developing machine learning models that improve the accuracy and robustness of these predictions. 
```

Comment:
"address" sounds like we are going to provide a full solution to these challenges and somehow solve the entire problem, might need to rephrase to tone it down a bit.

Reply:
```suggestion
Building upon the baseline established in~\citet{p9_paper}, this thesis aims to explore approaches for tackling the challenges in predicting major oxide compositions from \gls{libs} data. We develop machine learning models that seek to enhance the accuracy and robustness of these predictions.
```